### ISSUE TITLE
Tooltip with form control feedback

### ISSUE THREAD

**Issue by clemgrim** [twbs/bootstrap#16306]
Bootstrap tooltip does not shown when applied on `form-control-feedback` class.

This is caused by the folowing css rule `pointer-events: none;`

[in less, line 401](https://github.com/twbs/bootstrap/blob/master/less/forms.less)

See my [fiddle](https://jsfiddle.net/f42znLst/1/)

**Comment by twbs-lmvtfy** [comment]
Hi @clemgrim!

You appear to have posted a live example (https://fiddle.jshell.net/f42znLst/1/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 52, column 13 thru line 52, column 112: The ```` aria-describedby ```` attribute must point to an element in the same document.
* line 57, column 13 thru line 57, column 112: The ```` aria-describedby ```` attribute must point to an element in the same document.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by clemgrim** [comment]
Sorry, I fixed this.

**Comment by mdo** [comment]
Unsure what we can do for you here. Those icons weren't designed to be tooltipped, but that shouldn't explicitly be disallowed. However, from a UI perspective, you should put that kind of feedback on the type of error out in the open, not in a tooltip.

I don't think removing that line is an option—it'd cause some regressions for folks. If you really need the tooltip, I think you'll have to override that yourself per your fiddle.

**Comment by patrickhlauke** [comment]
Note that having a tooltip on those icons means that keyboard users (both sighted users and those using assistive technologies) will never be able to get to them/read them.

**Comment by cvrebert** [comment]
Yeah, you should probably put the message in a `.help-block`

**Comment by mdo** [comment]
Punting as a won't fix then. Thanks, fellas.
